src/services/ActivityCardService.js

```diff
@@ -1 +1,133 @@
-// placeholder file for Drive shenanigans woohoo!
+import {
+  checkFileExists,
+  writeFile,
+  makeDirectory,
+  moveFile,
+  deleteFile,
+} from './routes/Local';
+import { MAINDIRECTORY } from './constants';
+import { AccessToken } from './models';
+import { Buffer } from 'buffer';
+import axios from 'axios';
+import { DRIVE_API_URLS } from './config.json';
+
+const ActivityCardService = {
+  // All APIs for ActivityCards should be here
+  // Don't call RNFS directly here!
+  // Instead, call a function from Local.js :~]
+
+  /**
+   * Returns array of file paths for this month's featured activity cards.
+   * @return {String[]}
+   */
+  getFeaturedActivityCards: async function () {
+    try {
+      //Retrieve the last week from browser, convert to ISO for use in query
+      const weekAgo = new Date(
+        Date.now() - 8 * 24 * 60 * 60 * 1000,
+      ).toISOString();
+
+      //Set up GET url, query, and path to Featured Card directory
+      var downloadUrl =
+        DRIVE_API_URLS.SEARCH_FILES + DRIVE_API_URLS.SEARCH_PARAMETERS;
+      var path = MAINDIRECTORY + '/FeaturedActivityCards/';
+      const searchQuery =
+        "modifiedTime >= '" +
+        weekAgo +
+        "' and mimeType contains 'image/' and fullText contains 'ACTVT'";
+      const params = {
+        q: searchQuery,
+        fields:
+          'files(kind,driveId,mimeType,id,name,teamDriveId,thumbnailLink,webContentLink,modifiedTime)',
+      };
+
+      //Retreive all Drive files meeting the params
+      const response = await axios.get(downloadUrl, { params }).catch(error => {
+        console.log('ERROR IN GETTING FEATURED ACTIVITY CARDS: ' + error);
+      });
+
+      //Access the Array of all files and set up path Array (to be returned)
+      const driveFiles = response.data;
+      const files_list = driveFiles.files;
+      var pathArr = [];
+
+      //Delete anything that may currently be in the Featured Cards directory, make the new path with no contents
+      if (files_list.length != 0 && (await checkFileExists(path))) {
+        await deleteFile(path);
+        await makeDirectory(path);
+      } else {
+        return;
```

Comment:
```suggestion
        return [];
```
Make sure all your return statements adhere to `@return {String[]}` and whatever calls `getFeaturedActivityCards` can handle the empty case.